types/wellknown/index.d.ts

```diff
@@ -1,7 +1,32 @@
 // Type definitions for wellknown 0.5
 // Project: https://github.com/mapbox/wellknown#readme
-// Definitions by: Yair Tawil <https://github.com/yairtawil>
+// Definitions by: Davide Scalzo <https://github.com/davodesign84>
```

Comment:
Is that indended change? Would adding a new author on the next line suffice?

Reply:
yup, good point